src/test/java/org/gridsuite/ds/server/controller/DynamicSimulationControllerTest.java

```diff
@@ -532,12 +537,12 @@ public void testStopEarly() throws Exception {
 
         // check result
         // Must have a cancel message in the stop queue
-        Message<byte[]> message = output.receive(1000, dsStoppedDestination);
+        Message<byte[]> message = output.receive(1000, dsCancelFailedDestination);
         assertThat(message.getHeaders())
                 .containsEntry(HEADER_RESULT_UUID, runUuid.toString())
-                .containsEntry(HEADER_MESSAGE, getCancelMessage(COMPUTATION_TYPE));
+                .containsEntry(HEADER_MESSAGE, getCancelFailedMessage(COMPUTATION_TYPE));
         // result has been deleted by cancel so not found
-        assertNotFoundResult(runUuid);
+        assertResultStatus(runUuid, status().isOk());
```

Comment:
this test doesn't do the thing it's saying it is anymore. The method name / comment are wrong now. 
since we are not fixing now the fact that we don't test the cancel on running computation we should either update the comment and names and add some detailed comment to explain why we are doing that.

Reply:
done